### ISSUE TITLE
kubelet无法启动

### ISSUE THREAD

**Issue by ningpengtao-coder** [sealerio/sealer#1447]
错误日志：
E0527 03:39:54.180060    1372 server.go:292] "Failed to run kubelet" err="failed to run Kubelet: misconfiguration: kubelet cgroup driver: \"cgroupfs\" is different from docker cgroup driver: \"system\"

复现：
升级到0.8.5安装kubernetes1.22以下base image。

与一下提交相关
 #1369

**Comment by sunzm4work** [comment]
可以在/etc/docker/daemon.json下修改docker的驱动为cgroupfs试试，或者反过来修改kubelet的

**Comment by allencloud** [comment]
Actually, the dockerd configuration and kubelet configuration are both stored in CloudImage, then we can update that in base images to keep both configuration consistent.

@kakaZhou719 Am I right ?

**Comment by kakaZhou719** [comment]
@ningpengtao-coder @allencloud , yes, we should keep the same cgroup driver between docker and kubelet. i will update the kubeadm of rootfs later.
